llvm/lib/SandboxIR/SandboxIR.cpp

```diff
@@ -612,7 +612,19 @@ void BranchInst::dump() const {
 
 LoadInst *LoadInst::create(Type *Ty, Value *Ptr, MaybeAlign Align,
                            Instruction *InsertBefore, Context &Ctx,
-                           bool IsVolatile, const Twine &Name) {
+                           const Twine &Name) {
+  llvm::Instruction *BeforeIR = InsertBefore->getTopmostLLVMInstruction();
```

Comment:
No need to re-implement the function. Just `return create(Ty, Ptr, Align, InsertBefore, /*IsVolatile=*/false, Ctx, Name);`